syntax = "proto3";

package estimator;

message Estimator {
  message Data {
    required float split_percent = 1;
    required string directory_pattern = 2;
  }

  message HiddenUnit {
    required int neurons = 1;
  }

  required int save_checkpoints_steps = 1;
  required string model_dir = 2;
  required int eval_throttle_secs = 3;
  required int train_steps = 4;
  required int batch_size = 5;
  repeated HiddenUnit hidden_units = 6;
}